### PR TITLE
Update French translations

### PR DESCRIPTION
**Pull request by alexislefebvre**
- Fix inconsistencies in the settings
- Fix questions: there is a space before the question mark in French

### CODE REVIEW

File: frontend/src/locales/fr.po [hunk 1]
```diff
@@ -273,23 +273,23 @@ msgstr "Archivé"
...
 
 #: src/dialog/account/remove.vue:10
 msgid "Are you sure you want to delete this account?"
-msgstr "Êtes-vous sûr de vouloir supprimer ce compte?"
+msgstr "Êtes-vous sûr de vouloir supprimer ce compte ?"
 
 #: src/dialog/file/delete.vue:10
 msgid "Are you sure you want to permanently delete this file?"
-msgstr "Êtes-vous sûr de vouloir supprimer définitivement ce fichier?"
+msgstr "Êtes-vous sûr de vouloir supprimer définitivement ce fichier ?"
```
Comment (lastzero) [inline frontend/src/locales/fr.po:292]: Why the blank space before ?

Comment (StephenBrown2) [reply frontend/src/locales/fr.po:292]: I was curious about this, so I googled it: https://iwillteachyoualanguage.com/learn/french/french-tips/french-punctuation

Comment (alexislefebvre) [reply frontend/src/locales/fr.po:292]: Yes, in French there is a space before question or exclamation mark. Actually a non-breaking space should be used.

Comment (lastzero) [reply frontend/src/locales/fr.po:292]: Interesting! Some Germans do it too, but it's wrong over here... there's even a Wikipedia article on this:

https://en.wikipedia.org/wiki/Plenken

**Comment by lastzero** [comment]
Merci :heart: 

I'll merge this. There are already new strings waiting for translation though. Just finished updating German.

**Pull request merged**